### ISSUE TITLE
Autoprefixer is now deprecated

### ISSUE THREAD

**Issue by pond** [twbs/bootstrap-rubygem#196]
In version 9.8.6 and 9.8.6.1 of the Autoprefixer gem, deprecation warnings are now printed to the console. Aside from the large amounts of console spam, the [gem documentation now makes it very clear](https://github.com/ai/autoprefixer-rails/wiki/Deprecated) that anyone using the gem is meant to move off it.

```
DEPRECATION WARNING: autoprefixer-rails was deprected. Migration guide:
https://github.com/ai/autoprefixer-rails/wiki/Deprecated (called from [...])
```

As a workaround for console spam, I've temporarily updated my `Gemfile` to explicitly require Autoprefixer (even though I don't use it) v9.8.5.

**Comment by aseroff** [comment]
Seems like unless this wrapper is refactored to use a different prefix library, rails applications are looking at a forced conversion from asset pipeline to webpacker, just to use babel? Or am I missing something?

**Comment by pond** [comment]
I don't know or particularly care. The JavaScript / "asset management" crowd seem to think up new, even more complicated ways to do simple things every day of the week; all I know is that Bootstrap started spewing warnings because it depends on Autoprefixer, which is now end-of-life.

It's Bootstrap pulling in the dependency, not our application. So I don't care what it uses instead; it just can't keep using a gem that's now been killed off.

**Comment by ilikepi** [comment]
We ran into this issue yesterday as well.  On further investigation, it looks like someone has offered to take over maintainership (ai/autoprefixer-rails#168).  In the meantime, we eliminated the warnings by locking autoprefixer-rails to version 9.8.5, which was the last release before the warnings were added.  The changelog mentions something about fixing an "`env` option" in 9.8.6, but it's not immediately clear what that is.  Otherwise all the more recent releases appear to have dealt with introducing the warnings.

Alternatively, it seems like it would be fairly trivial to monkeypatch the method that outputs the deprecation warning...looks like it's in `lib/autoprefixer-rails/processor.rb`.
